### ISSUE TITLE
[Keyboard Navigation-Data Explorer (iframe)] Keyboard focus indicator is not visible on expanded suggestion list “Empty array , Empty object ” under  “Editor pane”.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1706]
Bug copied from Azure cosmos DB : https://msdata.visualstudio.com/CosmosDB/_workitems/edit/527036

**URL:** https://ms.portal.azure.com/

**User Experience:**
For keyboard users  It is hard to recognize the element which contains current focus if visual focus indicator is not visible on any controls.

**Pre-Requisite:**
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

**Repro Steps:**

1. Open URL: https://ms.portal.azure.com/
2. Login via valid credentials
3. Navigate to "Azure Cosmos DB " from left pane and select it.
4. Navigate to any of the Azure Cosmos DB which is already created and select it.
5. Navigate to "Data Explorer" from the left and select it.
6. Navigate through all the controls under SQL API Section and select Documents item.
7. Navigate to "New Document" button and select it.
8. Navigate through the editor Pane.
9 .Navigate all the controls of "Editor pane”.
10. Verify the keyboard focus clearly visible or not on all controls.

**Actual Result:**
Keyboard focus indicator is not clearly visible on expanded list “Empty array , Empty object ” under “Editor pane”.

**Expected Result:**
Keyboard focus indicator should be clearly visible on expanded list “Empty array , Empty object ” under “Editor pane”, so that user can identify that which element contains current focus.

**Note:**
Similar issue exist with Chrome.
Similar issue exist for all suggestion list in editor pane.
Similar issue exist throughout the application 

**MAS Reference:**

MAS 2.4.7 - Focus Visible (05A)

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier. 

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug.

For more information please use this Link.
**Attachment** : [MAS2.4.7_empty.zip](https://github.com/microsoft/monaco-editor/files/3963785/MAS2.4.7_empty.zip)

**Comment by alexdima** [comment]
This is not an issue with the Monaco Editor because by default, the editor does render suggest widget focus:

![image](https://user-images.githubusercontent.com/5047891/70893521-8eb19600-1feb-11ea-81fe-fc25a4045468.png)

Please try to reproduce at https://microsoft.github.io/monaco-editor/ before filing such issues here. In this case, it appears that the embedder has a CSS rule that prohibits the suggest widget from rendering normally. Thank you!
